src/functions-and-arrays.js

```diff
@@ -1,19 +1,67 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(num1, num2) {
+  if(num1 > num2){
+    return num1
+  }
+  else if(num2 > num1){
+    return num2
+  }
+  else{return num1}
```

Comment:
You could simplify this function by instead of else if on the second member, doing just else { return num2}. Logically speaking, if num1 is not bigger then num2, it can only be smaller or equal, and seeing as when it's equal it doesn't matter what you return... good job overall, though :)